handler/handler_test.go

```diff
@@ -37,33 +42,36 @@ var (
 	errS3              = errors.New("test S3Upload error")
 	errVault           = errors.New("test Vault error")
 	errPsk             = errors.New("test PSK error")
+	errDataset         = errors.New("test DatasetAPI error")
 )
 
 var originalCreatePSK = handler.CreatePSK
 
 var ctx = context.Background()
 
-func TestInstanceCompleteHandler_Handle(t *testing.T) {
+// TODO uncomment and finish implementing test after the whole functionality for the handler is implemented
```

Comment:
I'm not sure unit tests are the right place to be testing the Handler, after that convo we had with James et al it seemed the general idea was to focus on component tests for the major functionality, unit tests just for code snippets

Reply:
ok, the test was originally here and I commented it - but maybe we want to remove it completely?

Reply:
Yeah it may have been started before we had that conversation/came as part of the code generation. I think we should remove it.